src/styles/blocks/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.star {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
BEM

Reply:
⚠️ didn't fixed

Reply:
Sorry,I don't understand why it doesn't follow BEM's rules. Why I can't set width and height for element of stars block. It isn't an independent block, otherwise how I can set a size for the background-image corectly?

Reply:
Sorry, this is my mistake. Everything is fine here, I thought it was a block.